### DISCUSSION TITLE
I set batch count = 2, how to save the generated 2 images？

### DISCUSSION THREAD

**Discussion by supwang** [comfyanonymous/ComfyUI#635] [Q&A]
Hi, All:

I set batch count = 2, how to save the generated 2 images？
There is only one image that I can find in node of "Save Image".

**Comment by ltdrdata** [comment]
SaveImage allows batch images.

**Reply by supwang** [reply]
Thank you.
But I can see only one image. The image of first batch will be covered by the second image.

**Comment by Zuellni** [comment]
Batch count generates the images sequentially so only 1 will be visible at a time. To generate more at the same time you have to increase `batch_size` in the `Empty Latent Image` node.

**Reply by supwang** [reply]
Solved.
Thank you.
